### PR TITLE
feat(FTL-7654): added update to SDK to support the `Truecaller` flow

### PR DESCRIPTION
**Pull request by oleksiipiliugin**
Implementation of the signin flow with the [Truecaller](https://docs.truecaller.com/truecaller-sdk/) profile/token.

Logic is the same as for sigin passwordless with phone number, but without sending OTP to the user.
With this new feature, we are able to create a new user in the Cognito pool or login if it exists.
The main changes are in `sdk/core` and `lib/user-management`. Also, support for new endpoints was added for the internal key storage service.

### CODE REVIEW

**Review by yaroslava-kurash — COMMENTED** [review]
Shouldn't versions of `@affinidi/wallet-browser-sdk`  `@affinidi/wallet-expo-sdk`  `@affinidi/wallet-node-sdk` and `@affinidi/wallet-react-native-sdk` be updated too as they use new version of `@affinidi/wallet-core-sdk`

**Review by yaroslava-kurash — APPROVED** [review]

File: sdk/core/test/integration/index.test.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import './env'
 import { useNodeFetch } from '@affinidi/platform-fetch-node'
 useNodeFetch()
 
-require('./services')
-require('./AffinityWallet.test')
-require('./NetworkMember.test')
+// require('./services')
```
Comment (linn-affinidi) [inline sdk/core/test/integration/index.test.ts:7]: forgot to enable back?

File: sdk/core/test/integration/otp/core.test.ts [hunk 2]
```diff
@@ -643,3 +658,79 @@ parallel('CommonNetworkMember [OTP]', () => {
     })
   })
 })
+
+// describe('CommonNetworkMember [TrueCaller]', () => {
```
Comment (Volland) [inline sdk/core/test/integration/otp/core.test.ts:662]: @oleksiipiliugin why we comment it out ?

Comment (Volland) [reply sdk/core/test/integration/otp/core.test.ts:662]: ah we probably need to deploy awb first

**Review by Volland — APPROVED** [review]

**Review by yaroslava-kurash — APPROVED** [review]

**Pull request merged**